Alarmify/Modules/Edit/Subviews/SoundPickerView.swift

```diff
@@ -7,27 +7,26 @@
 
 import SwiftUI
 
-struct SoundSelectorView: View {
+struct SoundPickerView: View {
     @EnvironmentObject var alarmdata: AlarmData
+    @Environment(\.colorScheme) var colorScheme
     var body: some View {
-        
         NavigationStack {
-            
-            Picker(selection: $alarmdata.CurrentSound) {
-                ForEach(alarmdata.Sounds, id: \.self) {
+            Picker(selection: $alarmdata.currentSound) {
+                ForEach(alarmdata.sounds, id: \.self) {
                     Text($0)
                 }
             } label: {
                 Text("Sound")
-                    .foregroundStyle(.white)
+                    .foregroundStyle(colorScheme == .dark ? .white: .black)
```

Comment:
There are native dynamic colors for this like Color.label, .background, .primary, .secondary

https://developer.apple.com/design/human-interface-guidelines/color#Specifications
https://developer.apple.com/documentation/SwiftUI/Color#getting-standard-colors

Reply:
Thanks, Fixed this!
This is the new thing which I learned today